frontend/src/components/EmailConfirmation.tsx

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+import { useParams } from "react-router-dom";
+
+const EmailConfirmation = () => {
+    const { id } = useParams<{ id: string }>();
+
+    axios.put(`http://localhost:3000/users/${id}`, {confirmed: true})
```

Comment:
Put all the network requests inside `useEffect` hook to prevent side-effects and bugs.